yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We don't need yarn for api

Reply:
I tried yarn install to maybe install some missing javascript for the rswag but it didn't work